### PR TITLE
New page - "Creating agendas"

### PR DESCRIPTION
**Pull request by ktoddbrown**
### Summary

This create an 'agenda create' guide for the Collaboration section per Issue #3957.

A well structured meeting can leave people feeling connected and appreciated as well as providing accountability and developing shared strategies. Right now the collaboration guide is missing guidance around design principles of agenda creation and example templates of common academic meetings.

This new section that will cover:

- goals of different sections of an agenda
- design of agenda items with specific purposes
- examples of common academic agendas

### List of changes proposed in this PR (pull-request)

* new file `website/collaboration/creating-agendas.md`

### What should a reviewer concentrate their feedback on?

- [ ] Is this content useful?
- [ ] Is this in the appropriate location?
- [ ] Everything looks ok?

### Acknowledging contributors

- [ ] The following people should be added to the [table of contributors](https://github.com/the-turing-way/the-turing-way/blob/main/README.md#contributors) in the README file: @ktoddbrown

### CODE REVIEW

File: book/website/collaboration/creating-agendas.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+- **M**inutes: Review the minutes of the previous meeting can remind people of what was discussed and decided at the last meeting. This is particuarlly useful for meetings that are infrequent.
+- **I**nformation: Make any FYI announcements. In some meetings the call for information can be a dedicated agenda item (see next section).
+- **N**ext call: When are you meeting next? Now is a good time to pull out your calendars and pick a time! The end of the meeting folks may be running off to their next appointments so getting this out of the way at the start ensures that everyone is acoomindated.
+
+Part of housekeeping is tending to the emotional wellbeing of the group.
+Providing a space for each person to check in with "How are you showing up?" can help people show up as a complete person, provide an opportunity for human-to-human connections, and set asside outside concerns to focus on the meeting.
+Folks sometimes have a hard time naming emotions and it can be helpful to link in a [feelings wheel](https://feelingswheel.com/) to help folks identify what resonates with them in the moment.
```
Comment (aleesteele) [inline book/website/collaboration/creating-agendas.md:32]: I've never seen this before - a really interesting project!

File: book/website/collaboration/creating-agendas.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+## The beginning
+
+The begining of anything is critical.
+Beginnings of agendas are connecting points, bridges from our everyday live into our collaborative working spaces.
+The start of any meeting happens when the first person logs into the call or walks into the room.
+Often the first person to arrive can set the tone through their greetings of subsequent members.
+
+The begining of an agenda is commonly focused on adminstration and housekeeping tasks.
+There are several conventions you can build off of here.
+[Sociocracy](https://www.sociocracyforall.org/meeting-format/) uses ADMIN to open their meetings.
```
Comment (aleesteele) [inline book/website/collaboration/creating-agendas.md:22]: This is a great format! I'm wondering if you could cite other frameworks (not describe them if you're not familiar of course), but rather give folks other options outside of the excellent one you describered for Sociocracy. We're all about plurality at The Turing Way - and this sounds like a great mode of allow people to choose what might work for them! For example, the RAPID framework, LEAN, etc. https://blog.pigeonholelive.com/how-to-make-meetings-more-effective

File: book/website/collaboration/creating-agendas.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+This is the part that most people think of when they hear 'agenda'.
+Generally items in this section will fall into three categories if the group is trying to come to a shared decision:
+
+1) Sensemaking - What is the context and background that frames a decision? This is a shared experience. One person in the group might have a perfect understanding of the background and context, but if that is not shared and commonly understood, then as a collective the group has not engaged in sensemaking.
+2) Formulation or brainstorming - What are ways we could address the decision in front of us? This is where we get to be expansive and throw ideas around. The goal of this agenda is not to get it 'right' but to gather all possible ideas about something.
+3) Decision - What is the decision we are making as a group? This seems like an obivious topic but it's very easy for a group to leave a meeting with different understandings of what was decided. If a decision is made, say so, ideally in writing.
+
+This is also a good place to check in on sub-circles or individual to-do items.
+A committee has never done anything in the history of time, and a person has never done anything in issolation.
```
Comment (aleesteele) [inline book/website/collaboration/creating-agendas.md:53]:
```suggestion
A committee has never done anything in the history of time, and a person has never done anything in isolation.
```
Maybe a clarification on my end: what do you mean that a committee has never done anything in the history of time? Is this a theoretical synario?

File: book/website/collaboration/creating-agendas.md (hunk 2 above)
Comment (ktoddbrown) [reply book/website/collaboration/creating-agendas.md:22]: What you have linked in here are less meeting structures and more collaboration arches in my mind. A lot of the team science literature focuses at this 'arch' level and I've seen less specifically about agenda construction. Long and short of it, this sociocracy guide has probably the most concrete guidance I've come across. I would love to see other people build off of this however!

TLDR I'm probably not the one to extend this but would be delighted to see other people build off if it here!

File: book/website/collaboration/creating-agendas.md (hunk 3 above)
Comment (ktoddbrown) [reply book/website/collaboration/creating-agendas.md:53]: Someone has to implement the strategy that the group comes up with. I often see busy researchers discuss and strategize without ever deciding who is going to write the paper, run the experiment, or code the model. Everyone just expects someone else to step in! By specifically naming who is doing what (defining roles) you both recognize folks for the work they are doing and make sure that the group strategy/decision gets implemented.

Is there a clearer way to say this that I'm missing?

File: book/website/collaboration/creating-agendas.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+Timing for these agenda items can be tricky.
+As a new facilitator/chair/lead you will almost certainly put too much on your first few agendas, that's ok.
+Timings are easier with experience.
+As a general rule of thumb people at the meeting are there because they have something to offer, so agenda items need to have enough time for everyone to speak to the item at hand. 
+Generally speaking 2-3 minutes per person seems to be enough to hear from everyone once on a topic, and topics that are particuarlly heated or with new groups may need to double this timing for a second round.
+It's not uncommon for meetings to only have 2 or 3 topics on an hour long agenda, construct your agenda carefully.
+
+## The end
+
+The end of an agend is a place to lay the groundworks for the next meeting and reflective improvement of how the meeting went.
```
Comment (aleesteele) [inline book/website/collaboration/creating-agendas.md:66]:
```suggestion
The end of an agenda is a place to lay the groundworks for the next meeting and reflective improvement of how the meeting went.
```

File: book/website/collaboration/creating-agendas.md [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+This is also a good place to check in on sub-circles or individual to-do items.
+A committee has never done anything in the history of time, and a person has never done anything in issolation.
+The work that you are doing is done by the members of your group.
+By taking time to track progress you are both recognizing who is doing that work and providing social accountability for that working getting done.
+
+Timing for these agenda items can be tricky.
+As a new facilitator/chair/lead you will almost certainly put too much on your first few agendas, that's ok.
+Timings are easier with experience.
+As a general rule of thumb people at the meeting are there because they have something to offer, so agenda items need to have enough time for everyone to speak to the item at hand. 
+Generally speaking 2-3 minutes per person seems to be enough to hear from everyone once on a topic, and topics that are particuarlly heated or with new groups may need to double this timing for a second round.
```
Comment (aleesteele) [inline book/website/collaboration/creating-agendas.md:61]:
```suggestion
Generally speaking 2-3 minutes per person seems to be enough to hear from everyone once on a topic, and topics that are particularly heated or with new groups may need to double this timing for a second round.
```

File: book/website/collaboration/creating-agendas.md [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+It's not uncommon for meetings to only have 2 or 3 topics on an hour long agenda, construct your agenda carefully.
```
Comment (aleesteele) [inline book/website/collaboration/creating-agendas.md:62]:
```suggestion
It's not uncommon for meetings to only have 2 or 3 topics in an hour long agenda, construct your agenda carefully.
```

File: book/website/collaboration/creating-agendas.md [hunk 7]
```diff
@@ -0,0 +1,111 @@
...
+
+## The end
+
+The end of an agenda is a place to lay the groundworks for the next meeting and reflective improvement of how the meeting went.
+
+Groundworks for the next meeting includes both personal tasks and collectively give input on what the next meeting agenda might look like.
+It can be a good idea to take a pause and let people write down and share the todo or action items they are taking away from the meeting.
+This gives individuals a chance to check in with the group before they go off and do a task.
+By asking "What should we be talking about in the next 2-4 calls?" you give everyone the opertunity to stear the direction of the team.
```
Comment (aleesteele) [inline book/website/collaboration/creating-agendas.md:71]:
```suggestion
By asking "What should we be talking about in the next 2-4 calls?" you give everyone the opportunity to stear the direction of the team.
```

File: book/website/collaboration/creating-agendas.md [hunk 8]
```diff
@@ -0,0 +1,111 @@
...
+
+This is also a good place to check in on sub-circles or individual to-do items.
+A committee has never done anything in the history of time, and a person has never done anything in issolation.
+The work that you are doing is done by the members of your group.
+By taking time to track progress you are both recognizing who is doing that work and providing social accountability for that working getting done.
+
+Timing for these agenda items can be tricky.
+As a new facilitator/chair/lead you will almost certainly put too much on your first few agendas, that's ok.
+Timings are easier with experience.
+As a general rule of thumb people at the meeting are there because they have something to offer, so agenda items need to have enough time for everyone to speak to the item at hand.
```
Comment (aleesteele) [inline book/website/collaboration/creating-agendas.md:60]:
```suggestion
As a general rule of thumb, people at the meeting are there because they have something to offer - so agenda items need to have enough time for everyone to speak to the item at hand. 
```
